### PR TITLE
Staging/home

### PR DESCRIPTION
**Pull request by akong00**
initial home page

### CODE REVIEW

File: src/components/Home/index.jsx [hunk 1]
```diff
@@ -1,8 +1,517 @@
...
+import foregroundBush from 'assets/svgs/foreground_bushes.svg';
+import car from 'assets/svgs/HackIllinois_Website_car.svg';
 
+const Container = Styled.div`
+  position: relative;
+  overflow-x: hidden;
+  background-image: url(${bg});
+  background-position: left top;
+  background-size: cover;
+  /* @media(max-width: 1000px){
```
Comment (brianstrauch) [inline src/components/Home/index.jsx]: Don't commit commented code! BIG NO NO

Comment (brianstrauch) [reply src/components/Home/index.jsx]: 😡

File: src/components/Home/index.jsx [hunk 2]
```diff
@@ -1,8 +1,517 @@
...
+    grid-area: 3/3/6/5;
+  }
+
+  @media(max-width: 900px) {
+    padding: 0 40px;
+    grid-area: 3/1/4/2;
+  }
+`;
+
+const DescTitle = Styled.div`
```
Comment (brianstrauch) [inline src/components/Home/index.jsx]: What does desc mean? Description? If so write the whole word

File: src/components/Home/index.jsx [hunk 3]
```diff
@@ -1,8 +1,517 @@
...
+const ForegroundBush = Styled.img`
+  position: relative;
+  z-index: 5;
+  width: 90%;
+  margin: -2.5% 5%;
+`;
+
+const Car = Styled.img.attrs(props => {
+  let p = props.position - 80;
+  return({
```
Comment (brianstrauch) [inline src/components/Home/index.jsx]: Don't think the parentheses are necessary here

File: src/components/Home/index.jsx [hunk 4]
```diff
@@ -1,8 +1,517 @@
...
+    let sh = document.body.scrollHeight;
+    // console.log(h, st, sh);
+    this.setState({scrollPos: st / (sh - h) * 100});
+  }
+
   render() {
-    return <p>/home</p>;
+    const { FAQState, scrollPos }= this.state;
+
+    const Descriptions = [
```
Comment (brianstrauch) [inline src/components/Home/index.jsx]: constants should be UPPER_SNAKE_CASE

Comment (brianstrauch) [reply src/components/Home/index.jsx]: I really like how you grouped all the text into objects here!! You might want to put all of these constants into a separate JS file though. They account for a lot of lines and detract from the CSS/JSX

**Review by brianstrauch — CHANGES_REQUESTED** [review]
Looks really good, let's make all the assets lower_camel_case and stick them in the assets folder, grouped by page, so assets/home/example_asset.svg
Not a fan of the svgs folder since pretty much everything is an SVG
Also assets starting with hackillinois_website are redundant

Looks really good, I want to play around with it before I approve it

**Review by brianstrauch — APPROVED** [review]

**Review by Aitori — APPROVED** [review]

**Pull request merged**